### DISCUSSION TITLE
Introducing Skeleton Labs!

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#604] [Announcements]
Hey everyone,

Chris here from the Skeleton crew. I am thrilled to finally be able to pull back the curtain and reveal **Skeleton Labs** - a new company and organization for Skeleton formed by myself and Trey Weir. We've provided a new website that covers everything you need to know - the company mission, services, and even a brief Q&A. Find that below.

https://skeletonlabs.co/

## History

To provide some background, Skeleton was originally created by [myself](https://github.com/endigo9740) and [Thomas Jespersen](https://github.com/thomasbjespersen) under the leadership of Trey at [Brain & Bones](https://www.brainandbonesllc.com/). In February of this year, we started work on a new UI component system for the company that would pair the best of what Svelte and Tailwind had to offer. This closed source project was the first iteration of what would become Skeleton.

Fast forward to August of this year, the project was open sourced and met with an overwhelmingly positive reception from the Svelte community. Even now we're averaging around 100 stars on GitHub/month and nearly 500 weekly downloads on NPM with only the bare minimum of promotion. Given the resounding success of the Skeleton library, we've made the decision to move Skeleton from a side project under Brain & Bones to a dedicated new company and organization. This is Skeleton Labs.

This is the next major step for the project and opens the door to many new possibilities. To better understand what this will provide, we recommend viewing the new [Skeleton Labs](https://skeletonlabs.co/) website.

## Migration

With the launch of Skeleton Labs, we've introduced a new NPM organization, which means a newly scoped NPM package for the Skeleton library. We've provided a quick three step process for migrating your existing projects. This is required if you wish to continue receiving updates going forward, so please do not skip this step!

- https://github.com/skeletonlabs/skeleton/discussions/603

## Sponsors

Skeleton Labs opens the door to direct sponsorship of the project. We will be launching new sponsorship programs through GitHub Sponsors and similar services very soon. Whether you're a current Brain & Bones sponsor or wish to begin sponsoring the project, we encourage you to keep an eye out for our upcoming announcements.

## Wrapping Up

Finally, if we missed anything, let us know in the comment section below or reach out in our community Discord (via the #general channel). I will be happy to respond and answer any questions you may have. That said, I'm very excited about the future of Skeleton, and hope we can continue to provide updates that meet your highest expectations. I appreciate your continued support.

Cheers,
Chris Simmons

**Comment by dslatkin** [comment]
Congratulations! Greatly appreciate the transparency and clear communication. Skeleton has a bright future, really looking forward to a v1. 😄

**Reply by endigo9740** [reply]
Absolutely! I'm super pumped at what the future holds for both the company and Skeleton itself. There's lots of hard work to be done, but this opens the door to a lot of possibilities!

I didn't say it above, but just so everyone knows - the v1.0 launch of Skeleton is still top priority. We're making good progress on our [roadmap](https://github.com/skeletonlabs/skeleton/discussions/503), so I'm optimistic this isn't too far out in front of us! :)
